### PR TITLE
go-ethereum 1.6.6-stable

### PR DESCRIPTION
**Pull request by krischik**
###### Tested on
macOS 10.x
Xcode 8.x

###### Verification 
Have you
- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by l2dy** [comment]
Build phrase runs `make all` by default (`build.cmd`+`build.target`), you don't need to set `build.cmd`. See https://guide.macports.org/#reference.phases.build.
Also, you should
* use spaces instead of tabs.
* use our standard modeline `# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4`

File: finance/go-ethereum/Portfile [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+PortSystem 1.0
+
+name				go-ethereum
+version				1.6.6
+revision			0
+categories			finance crypto
+license				GPL-3
+platforms			darwin
+maintainers			openmaintainer krischik
```
Comment (raimue) [inline finance/go-ethereum/Portfile]: The maintainer option should now be written like this to include both emails and GitHub usernames:
```
maintainers {krischik @krischik} openmaintainer
```

File: finance/go-ethereum/Portfile [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	xinstall -d ${destroot}${prefix}/bin
+	xinstall -m 755 ${worksrcpath}/build/bin/abigen		${destroot}${prefix}/bin/abigen
+	xinstall -m 755 ${worksrcpath}/build/bin/bootnode	${destroot}${prefix}/bin/bootnode
+	xinstall -m 755 ${worksrcpath}/build/bin/evm		${destroot}${prefix}/bin/evm
+	xinstall -m 755 ${worksrcpath}/build/bin/faucet		${destroot}${prefix}/bin/faucet
+	xinstall -m 755 ${worksrcpath}/build/bin/geth		${destroot}${prefix}/bin/geth
+	xinstall -m 755 ${worksrcpath}/build/bin/puppeth	${destroot}${prefix}/bin/puppeth
+	xinstall -m 755 ${worksrcpath}/build/bin/rlpdump	${destroot}${prefix}/bin/rlpdump
+	xinstall -m 755 ${worksrcpath}/build/bin/swarm		${destroot}${prefix}/bin/swarm
+	xinstall -m 755 ${worksrcpath}/build/bin/wnode		${destroot}${prefix}/bin/wnode
```
Comment (raimue) [inline finance/go-ethereum/Portfile]: The destroot phase could be simplified like this:
```
destroot {
    xinstall -m 755 -W ${worksrcpath}/build/bin abigen bootnode ... ${destroot}${prefix}/bin/
}
```

File: finance/go-ethereum/Portfile [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+license				GPL-3
+platforms			darwin
+maintainers			openmaintainer krischik
+description			Official golang implementation of the Ethereum protocol.
+long_description	Official golang implementation of the Ethereum protocol.
+
+homepage			https://github.com/ethereum/${name}
+master_sites		${homepage}/archive/
+distfiles			v${version}.tar.gz
+distname			go-ethereum-${version}
```
Comment (raimue) [inline finance/go-ethereum/Portfile]: The github port group should be used to reduce the complexity of the Portfile. Documentation is in the header comments in the [github port group](https://github.com/macports/macports-ports/blob/master/_resources/port1.0/group/github-1.0.tcl) or look for other ports using `github.setup`.

**Comment by pmetzger** [comment]
@krischik Would you be in a position to apply @raimue's suggested fixes to the Portfile? It would be good for us to finally either commit this or close it.

**Comment by pmetzger** [comment]
Since @krischik has not replied to the comments on his pull request, and it has been seven months, I'm inclined to close this.

**Comment by pmetzger** [comment]
Closing. @krischik, feel free to re-open if you wish.

**Pull request closed without merging**